### ISSUE TITLE
Phantomjs raises erroneous SyntaxError on valid minified js file

### ISSUE THREAD

**Issue by jeantil** [ariya/phantomjs#12867]
Trying to load https://openoox.com fails to load the website correctly, all screenshots are blank. 

Running phantomjs 1.9.8 (on mac os x, homebrew install) with the following arguments: 
````sh
phantomjs --webdriver=127.0.0.1:8000 --webdriver-loglevel=DEBUG --remote-debugger-port=9000 --debug=yes --ignore-ssl-errors=true --ssl-protocol=any --web-security=false 
````
And executing the following commands (requires [jq](http://stedolan.github.io/jq/)) : 
````sh
SESSIONID=`curl -s -X POST -d '{"requiredCapabilities":{},"desiredCapabilities":{}}' localhost:8000/session | jq -r '.sessionId'`
curl -s -X POST -d '{"url":"https://openoox.com"}' localhost:8000/session/$SESSIONID/url | jq '.'
````
I get the following relevant logs :
````
[DEBUG - 2015-01-08T13:49:51.587Z] Session [f4807a10-973c-11e4-9526-e361d3dfcbac] - page.onResourceReceived - {"contentType":"application/x-javascript","headers":[{"name":"Content-Type","value":"application/x-javascript"},{"name":"Last-Modified","value":"Fri, 24 Oct 2014 00:16:08 GMT"},{"name":"Vary","value":"Accept-Encoding"},{"name":"ETag","value":"W/\"54499a48-14915\""},{"name":"Expires","value":"Thu, 31 Dec 2037 23:55:55 GMT"},{"name":"Cache-Control","value":"max-age=315360000, public"},{"name":"Server","value":"NetDNA-cache/2.2"},{"name":"X-Cache","value":"HIT"},{"name":"Content-Encoding","value":"gzip"}],"id":5,"redirectURL":null,"stage":"end","status":200,"statusText":"OK","time":"2015-01-08T13:49:51.587Z","url":"https://code.jquery.com/jquery-2.1.1.min.js"}
[DEBUG - 2015-01-08T13:49:51.601Z] Session [f4807a10-973c-11e4-9526-e361d3dfcbac] - page.onResourceReceived - {"contentType":"application/x-javascript","headers":[{"name":"Content-Type","value":"application/x-javascript"},{"name":"Cache-Control","value":"max-age=15552000, public"},{"name":"Content-Encoding","value":"gzip"},{"name":"Expires","value":"Sat, 12 Dec 2015 08:33:48 UTC"},{"name":"Last-Modified","value":"Fri, 12 Dec 2014 08:34:05 GMT"},{"name":"ETag","value":"\"7640dcb71c008060aee42de9013b3318\""},{"name":"Accept-Ranges","value":"bytes"},{"name":"Server","value":"AmazonS3"},{"name":"Age","value":"281967"},{"name":"X-Cache","value":"Hit from cloudfront"},{"name":"Via","value":"1.1 19325df3c9f68640dd7aea4d9b1fe2db.cloudfront.net (CloudFront)"},{"name":"X-Amz-Cf-Id","value":"0kSVjMy76HivYsfPfPR23P8gmcduItp6JbmJjupjdzWo-lPSVVwOEg=="}],"id":6,"redirectURL":null,"stage":"end","status":200,"statusText":"OK","time":"2015-01-08T13:49:51.601Z","url":"https://d1c0q3htqmcax5.cloudfront.net/js/app.js?v=1418400659115"}
[ERROR - 2015-01-08T13:49:51.617Z] Session [f4807a10-973c-11e4-9526-e361d3dfcbac] - page.onError - msg: SyntaxError: Parse error
[ERROR - 2015-01-08T13:49:51.617Z] Session [f4807a10-973c-11e4-9526-e361d3dfcbac] - page.onError - stack:

[DEBUG - 2015-01-08T13:49:51.618Z] Session [f4807a10-973c-11e4-9526-e361d3dfcbac] - page.onResourceRequested - {"headers":[{"name":"User-Agent","value":"Mozilla/5.0 (Macintosh; PPC Mac OS X) AppleWebKit/534.34 (KHTML, like Gecko) PhantomJS/1.9.8 Safari/534.34"},{"name":"Accept","value":"*/*"},{"name":"Referer","value":"https://openoox.com/"}],"id":7,"method":"GET","time":"2015-01-08T13:49:51.618Z","url":"https://www.google-analytics.com/analytics.js"}
[DEBUG - 2015-01-08T13:49:51.620Z] Session [f4807a10-973c-11e4-9526-e361d3dfcbac] - page.onResourceReceived - {"bodySize":25393,"contentType":"text/javascript","headers":[{"name":"Expires","value":"Thu, 08 Jan 2015 13:54:51 GMT"},{"name":"Last-Modified","value":"Thu, 13 Nov 2014 21:10:00 GMT"},{"name":"X-Content-Type-Options","value":"nosniff"},{"name":"Content-Type","value":"text/javascript"},{"name":"Vary","value":"Accept-Encoding"},{"name":"Content-Encoding","value":"gzip"},{"name":"Server","value":"Golfe2"},{"name":"Age","value":"5083"},{"name":"Cache-Control","value":"public, max-age=7200"},{"name":"Alternate-Protocol","value":"443:quic,p=0.02"}],"id":7,"redirectURL":null,"stage":"start","status":200,"statusText":"OK","time":"2015-01-08T13:49:51.620Z","url":"https://www.google-analytics.com/analytics.js"}
2015-01-08T14:49:51 [DEBUG] WebPage - updateLoadingProgress: 43
[DEBUG - 2015-01-08T13:49:51.621Z] Session [f4807a10-973c-11e4-9526-e361d3dfcbac] - _execFuncAndWaitForLoadDecorator - Page Loading in Session: true
[DEBUG - 2015-01-08T13:49:51.621Z] Session [f4807a10-973c-11e4-9526-e361d3dfcbac] - page.onResourceReceived - {"contentType":"text/javascript","headers":[{"name":"Expires","value":"Thu, 08 Jan 2015 13:54:51 GMT"},{"name":"Last-Modified","value":"Thu, 13 Nov 2014 21:10:00 GMT"},{"name":"X-Content-Type-Options","value":"nosniff"},{"name":"Content-Type","value":"text/javascript"},{"name":"Vary","value":"Accept-Encoding"},{"name":"Content-Encoding","value":"gzip"},{"name":"Server","value":"Golfe2"},{"name":"Age","value":"5083"},{"name":"Cache-Control","value":"public, max-age=7200"},{"name":"Alternate-Protocol","value":"443:quic,p=0.02"}],"id":7,"redirectURL":null,"stage":"end","status":200,"statusText":"OK","time":"2015-01-08T13:49:51.621Z","url":"https://www.google-analytics.com/analytics.js"}
2015-01-08T14:49:51 [DEBUG] WebPage - updateLoadingProgress: 100
[DEBUG - 2015-01-08T13:49:51.621Z] Session [f4807a10-973c-11e4-9526-e361d3dfcbac] - _oneShotCallback - onLoadFinished
[DEBUG - 2015-01-08T13:49:51.621Z] Session [f4807a10-973c-11e4-9526-e361d3dfcbac] - _execFuncAndWaitForLoadDecorator - onLoadFinished: success
2015-01-08T14:49:51 [DEBUG] WebPage - setupFrame ""
2015-01-08T14:49:51 [DEBUG] CookieJar - Saved ...
[DEBUG - 2015-01-08T13:49:51.632Z] Session [f4807a10-973c-11e4-9526-e361d3dfcbac] - page.onResourceRequested - {"headers":[{"name":"User-Agent","value":"Mozilla/5.0 (Macintosh; PPC Mac OS X) AppleWebKit/534.34 (KHTML, like Gecko) PhantomJS/1.9.8 Safari/534.34"},{"name":"Referer","value":"https://openoox.com/"},{"name":"Accept","value":"*/*"}],"id":8,"method":"GET","time":"2015-01-08T13:49:51.632Z","url":"https://www.google-analytics.com/r/collect?v=1&_v=j31&a=1598868858&t=pageview&_s=1&dl=https%3A%2F%2Fopenoox.com%2F&ul=fr-fr&de=UTF-8&dt=Openoox&sd=32-bit&sr=1440x900&vp=400x300&je=0&_u=MACAAIQAI~&jid=2077055289&cid=877111669.1420723175&tid=UA-33089832-1&_r=1&z=886008095"}
[DEBUG - 2015-01-08T13:49:51.721Z] Session [f4807a10-973c-11e4-9526-e361d3dfcbac] - _execFuncAndWaitForLoadDecorator - Page Loading in Session: false
[DEBUG - 2015-01-08T13:49:51.721Z] SessionReqHand - _SuccessHandler - status: success
2015-01-08T14:49:51 [DEBUG] HTTP Response - Status Code 200 OK
````

when connecting on localhost:9000 and running the inspector for https://openoox.com I can see an error on app.js: 
````
app.js:10SyntaxError: Parse error
````

However, the same website loads fine in recent chrome, firefox, safari and IE installations, the minified app.js file also parses just fine through esprima .

Here is the script I used to do the parsing using the node.js esprima module :
````js
var fs = require('fs'),
    esprima = require('esprima'),
    files = process.argv.splice(2);
files.forEach(function (filename) {
    'use strict';
    var content = fs.readFileSync(filename, 'utf-8'),
        parsed = esprima.parse(content, {});
    console.log(JSON.stringify(parsed, null, 2));
});
````
Run as :  
````sh
node esprima.js app.js
````
After saving app.js locally (beware the gzip encoding)

**Comment by JamesMGreene** [comment]
When I look at the [JS file](), it looks like you do have a ton of parse errors on line 10, starting around character/column 17681 (do a find for `</div>",j={`).  I'm thinking perhaps your script file is not properly encoded and/or not being served with its correct encoding (e.g. `utf-8`)...?

```js
j={"â’¶":"A",ï¼¡:"A",Ã€:"A",Ã:"A",Ã‚:"A",áº¦:"A",áº¤:"A",áºª:"A",áº¨:"A",Ãƒ:"A",Ä€:"A",Ä‚:"A",áº°:"A",áº®:"A",áº´:"A",áº²:"A",È¦:"A",Ç :"A",Ã„:"A",Çž:"A",áº¢:"A",Ã…:"A",Çº:"A",Ç:"A",È€:"A",È‚:"A",áº :"A",áº¬:"A",áº¶:"A",á¸€:"A",Ä„:"A",Èº:"A",â±¯:"A",êœ²:"AA",Ã†:"AE",Ç¼:"AE",Ç¢:"AE",êœ´:"AO",êœ¶:"AU",êœ¸:"AV",êœº:"AV",êœ¼:"AY","â’·":"B",ï¼¢:"B",á¸‚:"B",á¸„:"B",á¸†:"B",Éƒ:"B",Æ‚:"B",Æ:"B","â’¸":"C",ï¼£:"C",Ä†:"C",Äˆ:"C",ÄŠ:"C",ÄŒ:"C",Ã‡:"C",á¸ˆ:"C",Æ‡:"C",È»:"C",êœ¾:"C","â’¹":"D",ï¼¤:"D",á¸Š:"D",ÄŽ:"D",á¸Œ:"D",á¸:"D",á¸’:"D",á¸Ž:"D",Ä:"D",Æ‹:"D",ÆŠ:"D",Æ‰:"D",ê¹:"D",Ç±:"DZ",Ç„:"DZ",Ç²:"Dz",Ç…:"Dz","â’º":"E",ï¼¥:"E",Ãˆ:"E",Ã‰:"E",ÃŠ:"E",á»€:"E",áº¾:"E",á»„:"E",á»‚:"E",áº¼:"E",Ä’:"E",á¸”:"E",á¸–:"E",Ä”:"E",Ä–:"E",Ã‹:"E",áºº:"E",Äš:"E",È„:"E",È†:"E",áº¸:"E",á»†:"E",È¨:"E",á¸œ:"E",Ä˜:"E",á¸˜:"E",á¸š:"E",Æ:"E",ÆŽ:"E","â’»":"F",ï¼¦:"F",á¸ž:"F",Æ‘:"F",ê»:"F","â’¼":"G",ï¼§:"G",Ç´:"G",Äœ:"G",á¸ :"G",Äž:"G",Ä :"G",Ç¦:"G",Ä¢:"G",Ç¤:"G",Æ“:"G","êž ":"G",ê½:"G",ê¾:"G","â’½":"H",ï¼¨:"H",Ä¤:"H",á¸¢:"H",á¸¦:"H",Èž:"H",á¸¤:"H",á¸¨:"H",á¸ª:"H",Ä¦:"H",â±§:"H",â±µ:"H","êž":"H","â’¾":"I",ï¼©:"I",ÃŒ:"I",Ã:"I",ÃŽ:"I",Ä¨:"I",Äª:"I",Ä¬:"I",Ä°:"I",Ã:"I",á¸®:"I",á»ˆ:"I",Ç:"I",Èˆ:"I",ÈŠ:"I",á»Š:"I",Ä®:"I",á¸¬:"I",Æ—:"I","â’¿":"J",ï¼ª:"J",Ä´:"J",Éˆ:"J","â“€":"K",ï¼«:"K",á¸°:"K",Ç¨:"K",á¸²:"K",Ä¶:"K",á¸´:"K",Æ˜:"K",â±©:"K",ê€:"K",ê‚:"K",ê„:"K","êž¢":"K","â“":"L",ï¼¬:"L",Ä¿:"L",Ä¹:"L",Ä½:"L",á¸¶:"L",á¸¸:"L",Ä»:"L",á¸¼:"L",á¸º:"L",Å:"L",È½:"L",â±¢:"L",â± :"L",êˆ:"L",ê†:"L",êž€:"L",Ç‡:"LJ",Çˆ:"Lj","â“‚":"M",ï¼­:"M",á¸¾:"M",á¹€:"M",á¹‚:"M",â±®:"M",Æœ:"M","â“ƒ":"N",ï¼®:"N",Ç¸:"N",Åƒ:"N",Ã‘:"N",á¹„:"N",Å‡:"N",á¹†:"N",Å…:"N",á¹Š:"N",á¹ˆ:"N",È :"N",Æ:"N","êž":"N","êž¤":"N",ÇŠ:"NJ",Ç‹:"Nj","â“„":"O",ï¼¯:"O",Ã’:"O",Ã“:"O",Ã”:"O",á»’:"O",á»:"O",á»–:"O",á»”:"O",Ã•:"O",á¹Œ:"O",È¬:"O",á¹Ž:"O",ÅŒ:"O",á¹:"O",á¹’:"O",ÅŽ:"O",È®:"O",È°:"O",Ã–:"O",Èª:"O",á»Ž:"O",Å:"O",Ç‘:"O",ÈŒ:"O",ÈŽ:"O",Æ :"O",á»œ:"O",á»š:"O",á» :"O",á»ž:"O",á»¢:"O",á»Œ:"O",á»˜:"O",Çª:"O",Ç¬:"O",Ã˜:"O",Ç¾:"O",Æ†:"O",ÆŸ:"O",êŠ:"O",êŒ:"O",Æ¢:"OI",êŽ:"OO",È¢:"OU","â“…":"P",ï¼°:"P",á¹”:"P",á¹–:"P",Æ¤:"P",â±£:"P",ê:"P",ê’:"P",ê”:"P","â“†":"Q",ï¼±:"Q",ê–:"Q",ê˜:"Q",ÉŠ:"Q","â“‡":"R",ï¼²:"R",Å”:"R",á¹˜:"R",Å˜:"R",È:"R",È’:"R",á¹š:"R",á¹œ:"R",Å–:"R",á¹ž:"R",ÉŒ:"R",â±¤:"R",êš:"R","êž¦":"R",êž‚:"R","â“ˆ":"S",ï¼³:"S",áºž:"S",Åš:"S",á¹¤:"S",Åœ:"S",á¹ :"S",Å :"S",á¹¦:"S",á¹¢:"S",á¹¨:"S",È˜:"S",Åž:"S","â±¾":"S","êž¨":"S",êž„:"S","â“‰":"T",ï¼´:"T",á¹ª:"T",Å¤:"T",á¹¬:"T",Èš:"T",Å¢:"T",á¹°:"T",á¹®:"T",Å¦:"T",Æ¬:"T",Æ®:"T",È¾:"T",êž†:"T",êœ¨:"TZ","â“Š":"U",ï¼µ:"U",Ã™:"U",Ãš:"U",Ã›:"U",Å¨:"U",á¹¸:"U",Åª:"U",á¹º:"U",Å¬:"U",Ãœ:"U",Ç›:"U",Ç—:"U",Ç•:"U",Ç™:"U",á»¦:"U",Å®:"U",Å°:"U",Ç“:"U",È”:"U",È–:"U",Æ¯:"U",á»ª:"U",á»¨:"U",á»®:"U",á»¬:"U",á»°:"U",á»¤:"U",á¹²:"U",Å²:"U",á¹¶:"U",á¹´:"U",É„:"U","â“‹":"V",ï¼¶:"V",á¹¼:"V",á¹¾:"V",Æ²:"V",êž:"V",É…:"V",ê :"VY","â“Œ":"W",ï¼·:"W",áº€:"W",áº‚:"W",Å´:"W",áº†:"W",áº„:"W",áºˆ:"W",â±²:"W","â“":"X",ï¼¸:"X",áºŠ:"X",áºŒ:"X","â“Ž":"Y",ï¼¹:"Y",á»²:"Y",Ã:"Y",Å¶:"Y",á»¸:"Y",È²:"Y",áºŽ:"Y",Å¸:"Y",á»¶:"Y",á»´:"Y",Æ³:"Y",ÉŽ:"Y",á»¾:"Y","â“":"Z",ï¼º:"Z",Å¹:"Z",áº:"Z",Å»:"Z",Å½:"Z",áº’:"Z",áº”:"Z",Æµ:"Z",È¤:"Z","â±¿":"Z",â±«:"Z",ê¢:"Z","â“":"a",ï½:"a",áºš:"a",Ã :"a",Ã¡:"a",Ã¢:"a",áº§:"a",áº¥:"a",áº«:"a",áº©:"a",Ã£:"a",Ä:"a",Äƒ:"a",áº±:"a",áº¯:"a",áºµ:"a",áº³:"a",È§:"a",Ç¡:"a",Ã¤:"a",ÇŸ:"a",áº£:"a",Ã¥:"a",Ç»:"a",ÇŽ:"a",È:"a",Èƒ:"a",áº¡:"a",áº­:"a",áº·:"a",á¸:"a",Ä…:"a",â±¥:"a",É:"a",êœ³:"aa",Ã¦:"ae",Ç½:"ae",Ç£:"ae",êœµ:"ao",êœ·:"au",êœ¹:"av",êœ»:"av",êœ½:"ay","â“‘":"b",ï½‚:"b",á¸ƒ:"b",á¸…:"b",á¸‡:"b",Æ€:"b",Æƒ:"b",É“:"b","â“’":"c",ï½ƒ:"c",Ä‡:"c",Ä‰:"c",Ä‹:"c",Ä:"c",Ã§:"c",á¸‰:"c",Æˆ:"c",È¼:"c",êœ¿:"c",â†„:"c","â““":"d",ï½„:"d",á¸‹:"d",Ä:"d",á¸:"d",á¸‘:"d",á¸“:"d",á¸:"d",Ä‘:"d",ÆŒ:"d",É–:"d",É—:"d",êº:"d",Ç³:"dz",Ç†:"dz","â“”":"e",ï½…:"e",Ã¨:"e",Ã©:"e",Ãª:"e",á»:"e",áº¿:"e",á»…:"e",á»ƒ:"e",áº½:"e",Ä“:"e",á¸•:"e",á¸—:"e",Ä•:"e",Ä—:"e",Ã«:"e",áº»:"e",Ä›:"e",È…:"e",È‡:"e",áº¹:"e",á»‡:"e",È©:"e",á¸:"e",Ä™:"e",á¸™:"e",á¸›:"e",É‡:"e",É›:"e",Ç:"e","â“•":"f",ï½†:"f",á¸Ÿ:"f",Æ’:"f",ê¼:"f","â“–":"g",ï½‡:"g",Çµ:"g",Ä:"g",á¸¡:"g",ÄŸ:"g",Ä¡:"g",Ç§:"g",Ä£:"g",Ç¥:"g",É :"g","êž¡":"g",áµ¹:"g",ê¿:"g","â“—":"h",ï½ˆ:"h",Ä¥:"h",á¸£:"h",á¸§:"h",ÈŸ:"h",á¸¥:"h",á¸©:"h",á¸«:"h",áº–:"h",Ä§:"h",â±¨:"h",â±¶:"h",É¥:"h",Æ•:"hv","â“˜":"i",ï½‰:"i",Ã¬:"i",Ã­:"i",Ã®:"i",Ä©:"i",Ä«:"i",Ä­:"i",Ã¯:"i",á¸¯:"i",á»‰:"i",Ç:"i",È‰:"i",È‹:"i",á»‹:"i",Ä¯:"i",á¸­:"i",É¨:"i",Ä±:"i","â“™":"j",ï½Š:"j",Äµ:"j",Ç°:"j",É‰:"j","â“š":"k",ï½‹:"k",á¸±:"k",Ç©:"k",á¸³:"k",Ä·:"k",á¸µ:"k",Æ™:"k",â±ª:"k",ê:"k",êƒ:"k",ê…:"k","êž£":"k","â“›":"l",ï½Œ:"l",Å€:"l",Äº:"l",Ä¾:"l",á¸·:"l",á¸¹:"l",Ä¼:"l",á¸½:"l",á¸»:"l",Å¿:"l",Å‚:"l",Æš:"l",É«:"l",â±¡:"l",ê‰:"l",êž:"l",ê‡:"l",Ç‰:"lj","â“œ":"m",ï½:"m",á¸¿:"m",á¹:"m",á¹ƒ:"m",É±:"m",É¯:"m","â“":"n",ï½Ž:"n",Ç¹:"n",Å„:"n",Ã±:"n",á¹…:"n",Åˆ:"n",á¹‡:"n",Å†:"n",á¹‹:"n",á¹‰:"n",Æž:"n",É²:"n",Å‰:"n","êž‘":"n","êž¥":"n",ÇŒ:"nj","â“ž":"o",ï½:"o",Ã²:"o",Ã³:"o",Ã´:"o",á»“:"o",á»‘:"o",á»—:"o",á»•:"o",Ãµ:"o",á¹:"o",È­:"o",á¹:"o",Å:"o",á¹‘:"o",á¹“:"o",Å:"o",È¯:"o",È±:"o",Ã¶:"o",È«:"o",á»:"o",Å‘:"o",Ç’:"o",È:"o",È:"o",Æ¡:"o",á»:"o",á»›:"o",á»¡:"o",á»Ÿ:"o",á»£:"o",á»:"o",á»™:"o",Ç«:"o",Ç­:"o",Ã¸:"o",Ç¿:"o",É”:"o",ê‹:"o",ê:"o",Éµ:"o",Æ£:"oi",È£:"ou",ê:"oo","â“Ÿ":"p",ï½:"p",á¹•:"p",á¹—:"p",Æ¥:"p",áµ½:"p",ê‘:"p",ê“:"p",ê•:"p","â“ ":"q",ï½‘:"q",É‹:"q",ê—:"q",ê™:"q","â“¡":"r",ï½’:"r",Å•:"r",á¹™:"r",Å™:"r",È‘:"r",È“:"r",á¹›:"r",á¹:"r",Å—:"r",á¹Ÿ:"r",É:"r",É½:"r",ê›:"r","êž§":"r",êžƒ:"r","â“¢":"s",ï½“:"s",ÃŸ:"s",Å›:"s",á¹¥:"s",Å:"s",á¹¡:"s",Å¡:"s",á¹§:"s",á¹£:"s",á¹©:"s",È™:"s",ÅŸ:"s",È¿:"s","êž©":"s",êž…:"s",áº›:"s","â“£":"t",ï½”:"t",á¹«:"t",áº—:"t",Å¥:"t",á¹­:"t",È›:"t",Å£:"t",á¹±:"t",á¹¯:"t",Å§:"t",Æ­:"t",Êˆ:"t",â±¦:"t",êž‡:"t",êœ©:"tz","â“¤":"u",ï½•:"u",Ã¹:"u",Ãº:"u",Ã»:"u",Å©:"u",á¹¹:"u",Å«:"u",á¹»:"u",Å­:"u",Ã¼:"u",Çœ:"u",Ç˜:"u",Ç–:"u",Çš:"u",á»§:"u",Å¯:"u",Å±:"u",Ç”:"u",È•:"u",È—:"u",Æ°:"u",á»«:"u",á»©:"u",á»¯:"u",á»­:"u",á»±:"u",á»¥:"u",á¹³:"u",Å³:"u",á¹·:"u",á¹µ:"u",Ê‰:"u","â“¥":"v",ï½–:"v",á¹½:"v",á¹¿:"v",Ê‹:"v",êŸ:"v",ÊŒ:"v",ê¡:"vy","â“¦":"w",ï½—:"w",áº:"w",áºƒ:"w",Åµ:"w",áº‡:"w",áº…:"w",áº˜:"w",áº‰:"w",â±³:"w","â“§":"x",ï½˜:"x",áº‹:"x",áº:"x","â“¨":"y",ï½™:"y",á»³:"y",Ã½:"y",Å·:"y",á»¹:"y",È³:"y",áº:"y",Ã¿:"y",á»·:"y",áº™:"y",á»µ:"y",Æ´:"y",É:"y",á»¿:"y","â“©":"z",ï½š:"z",Åº:"z",áº‘:"z",Å¼:"z",Å¾:"z",áº“:"z",áº•:"z",Æ¶:"z",È¥:"z",É€:"z",â±¬:"z",ê£:"z",Î†:"Î‘",Îˆ:"Î•",Î‰:"Î—",ÎŠ:"Î™",Îª:"Î™",ÎŒ:"ÎŸ",ÎŽ:"Î¥",Î«:"Î¥",Î:"Î©",Î¬:"Î±",Î­:"Îµ",Î®:"Î·",Î¯:"Î¹",ÏŠ:"Î¹",Î:"Î¹",ÏŒ:"Î¿",Ï:"Ï…",Ï‹:"Ï…",Î°:"Ï…",Ï‰:"Ï‰",Ï‚:"Ïƒ"};
```

**Comment by JamesMGreene** [comment]
If I download the file to my hard drive (MacOSX), it looks fine in a good text editor like Sublime. However, opening it elsewhere (e.g. in Chrome's viewier), it looks like it would have parse errors.

**Comment by JamesMGreene** [comment]
> Maybe phantom should do the same ?

Agreed, and I would be surprised if `v2.0.0` (releasing in ~2 weeks) doesn't already do that.

**Comment by jeantil** [comment]
I'll try to build it from source to test (2 weeks away, preview binaries would be lovely, I dug in the ML history and the most recent build I could find was over 6 months old :( )

**Comment by jeantil** [comment]
The problem is fixed in Phantomjs 2.0, now to wait for the release :)
